NanoProd/crab/Run3_2023/SMHiggs.yaml

```diff
@@ -0,0 +1,7 @@
+config:
+  params:
+    sampleType: mc
+    era: Run3_2023
+
+#GluGluHToTauTau_M125: Missing
+#VBFHToTauTau_M125: Missing
```

Comment:
```suggestion
```
Missing samples can be traced in `Sample_exceptions.yaml`, so such comments are redundant.